acceptance/indy_acceptance/libraries/utils.py

```diff
@@ -0,0 +1,149 @@
+"""
+Created on Nov 9, 2017
+
+@author: khoi.ngo
+
+Containing all functions used by several test steps on test scenarios.
+"""
+from indy.error import IndyError
+from .constant import Color
+from . import constant
+from .result import Status
+
+
+def generate_random_string(prefix="", suffix="", size=20):
+    """
+    Generate random string .
+
+    :param prefix: (optional) Prefix of a string.
+    :param suffix: (optional) Suffix of a string.
+    :param size: (optional) Max length of a string (include prefix and suffix)
+    :return: The random string.
+    """
+    import random
+    import string
+    left_size = size - len(prefix) - len(suffix)
+    random_str = ""
+    if left_size > 0:
+        random_str = ''.join(random.choice(
+            string.ascii_uppercase + string.digits) for _ in range(left_size))
+    else:
+        print("Warning: Length of prefix and suffix more than %s chars"
+              % str(size))
+    result = str(prefix) + random_str + str(suffix)
+    return result
+
+
+def exit_if_exception(result):
+    """
+    If "code" is an exception then raise the "code".
+    Unless "code" is an exception then return the "code".
+    :param code: (optional) code that you want to check.
+    :return: "code" if it is not an exception.
+    """
```

Comment:
Update the docstring comment correspondingly to the parameter renaming.

Reply:
Will fix in next commit.